graalpy-embedded-pillow/pom.xml

```diff
@@ -0,0 +1,177 @@
+    </dependency>
+    <!-- The dependency on python-launcher is only required if the graalpy
+         plugin is used with packages -->
+    <dependency>
+      <groupId>org.graalvm.python</groupId>
+      <artifactId>python-launcher</artifactId>
+      <version>${graalpy.version}</version>
+    </dependency>
+    <!-- This dependency is required if packages should be embedded in
+         the Jar.  -->
+    <dependency>
+      <groupId>org.graalvm.python</groupId>
+      <artifactId>python-embedding</artifactId>
+        <version>${graalpy.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.graalvm.python</groupId>
+        <artifactId>graalpy-maven-plugin</artifactId>
+        <version>${graalpy-plugin.version}</version>
+        <executions>
+          <execution>
+            <configuration>
+              <!-- specify python packages as if used with pip -->
+              <packages>
+                <package>pillow</package>
+                <package>requests</package>
+              </packages>
+              <pythonHome>
+                <!-- java-like regular expression what file paths should be included into venv/home;
+                     default is all -->
+                <includes>
+                  <include>.*</include>
+                </includes>
+                <!-- java-like regular expression what file paths should be excluded from venv/home;
+                     default is none -->
+                <excludes>
+                </excludes>
+              </pythonHome>
+            </configuration>
+            <goals>
+              <goal>process-graalpy-resources</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.3.0</version>
+        <executions>
+          <execution>
+            <id>default-cli</id>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <executable>${env.JAVA_HOME}/bin/java</executable>
+          <arguments>
+            <argument>-classpath</argument>
+            <classpath />
+            <argument>${nativeimage.agentlib.argument}</argument>
+            <argument>${main.class}</argument>
+          </arguments>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>native</id>
+      <properties>
+        <nativeimage.agentlib.argument>-agentlib:native-image-agent=config-merge-dir=${project.build.outputDirectory}/META-INF/native-image/</nativeimage.agentlib.argument>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.graalvm.buildtools</groupId>
+            <artifactId>native-maven-plugin</artifactId>
+            <version>0.9.27</version>
```

Comment:
0.9.27 > 0.10.2

Reply:
I'm reluctant to change this in case it breaks!